### PR TITLE
 Add the requested changes to the landing page

### LINKED ISSUE #724: Remove the random coloured shapes from the landing page
**Is your feature request related to a problem? Please describe.**
It was discussed to remove the random coloured polygon shapes from the landing page to get a more professional look.

**Describe the solution you'd like**
Do the mentioned changes to the page

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #724 

## Goals

- Remove the random colored shapes from the landing page

## Approach

- Remove the random colored shapes from the landing page
- Change URL's for acdemix
- Change Onelive to OneLive
- Add a fixed height to the testimonials carousal 

### Screenshots

 
Screenshot from 2020-06-22 18-34-17

### Preview Link

https://pr-725-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/assets/css/global-custom.css [hunk 1]
```diff
@@ -561,3 +561,7 @@ body{
 .stats {
     font-size: 13px;
 }
+
+.carousel-item{
```
Comment (Piumal1999) [inline src/main/webapp/assets/css/global-custom.css]: A small problem occurs on the mobile view with this. Can you please look into that also
Screenshot_2020-06-22-19-20-07-004_com android chrome

**Comment by Piumal1999** [comment]
And also its better if we can change the heading of the pr. Because this pr fixes more issues other than the mentioned one.

File: src/main/webapp/assets/css/global-custom.css (hunk 1 above)
Comment (anjula-sack) [reply src/main/webapp/assets/css/global-custom.css]: Fixed! @Piumal1999

**Review by Piumal1999 — APPROVED** [review]
Well done! @anjula-sack

**Review by Gimhan-minion — APPROVED** [review]
Awesome @anjula-sack

**Pull request merged**